frontend/src/components/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FormControl, TextField, Button } from '@mui/material';
+import {
+  BorderForm,
+  RegistrationFormWrapper,
+  StyledSpan,
+  StyledSpanEnd
+} from 'components/LoginPage/styles';
+import {
+  TextFieldWrap,
+  WrapButtonAndText,
+  WrapH1
+} from 'components/ForgotPassword/styles';
+
+function LoginPage() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <h1>Login</h1>

```

Comment:
Typography component for h1 with variant prop

Reply:
done